torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8733,6 +8733,7 @@ def sample_inputs_scaled_mm(op_info, device, dtype, requires_grad, **kwargs):
     scale1 = make_scale((1,))
     scale2 = make_scale((1,))
     samples.append(SampleInput(mat1, mat2, scale1, scale2))
+    # TODO: Will remove this after oneDNN v3.6
```

Comment:
Please explain why?

Reply:
Updated.

Reply:
Don't you want to add (as part of separate PR perhaps) a test for e5m2 scaled_mm?